readme.md

```diff
@@ -80,244 +81,476 @@ As you're reviewing the app, try to fill in the blanks in the below Rails-to-Exp
+</details>
 
-- True or false: the environment variable for your Mongo database's location *must* be `MONGOLAB_URL`.
-- What's the value of `process.env.NODE_ENV` on Heroku? On your computer?
+<details>
+  <summary><strong>Why shouldn't you use `app.get` for `DELETE` routes?</strong></summary>
 
+  > (NEED ANSWER).)
 
-### [Added procfile and Mongolab URL for deployment: `db/connection.js`](https://www.github.com/ga-wdi-exercises/whenpresident/blob/02181c9/db/connection.js)
-![Added procfile and Mongolab URL for deployment, db/connection.js](_DIFFSHOTS/added-procfile-and-mongolab-url-for-deployment.db-connection-js.png)
+  <br/>
 
-#### Steps to get a Mongo database
+</details>
 
-1. `$ git push heroku master`
-- Go to www.mongolab.com and sign up / sign in
-- Create a new "Single Node" database with the "Sandbox" tier
-- Click on the database
-- Click "Users"
-- Create a new user. (This is *not* the user with which you logged in to Mongolab.) "User" in this context really means "an app that has access to your database". There's no need for security now; I used the username "test" with a password of "testerson".
-- Copy the "To connect using a driver" URL from the top of the Users page.
-- Set the URL as an environment variable called MONGOLAB_URL using `heroku config:set` as below, filling in the username and password you just created on the "Users" page. For example:
-    ```
-    $ heroku config:set MONGOLAB_URL=mongodb://test:testerson@ds015760.mlab.com:15760/yourappname
-    ```
-- `$ heroku run node db/seed.js`
-- `$ heroku open`
+![Delete 1](http://i.imgur.com/76mp0U4.png)
 
-## Conclusion
+> Again, **`method="post"`**. What's up with that?  
 
-1. What does `module.exports` do?
-- Why does `method="post"` even though we are updating (vs. creating) something?
-- Why do we use promises and callbacks when calling methods on a Mongoose model?
+### Steps
 
+1. In `index.js`, create a route that corresponds to our delete button.
+2. In it, use Mongoose to find and delete the candidate in question. (Hint: Refer to the Mongoose [lesson plan](https://github.com/ga-wdi-lessons/mongoose-intro#delete-5-min) or [documentation](http://mongoosejs.com/docs/api.html#query_Query-findOneAndRemove)).
 
-## Homework
+![Delete 2](http://i.imgur.com/1qF64Tf.png)
 
-Visit the [WhenPresident repo wiki](https://github.com/ga-wdi-exercises/whenpresident/wiki/Homework) for instructions.
+## Closing / Questions
 
-If you get stuck, feel free to review the solution branch (express-mongoose-solution):
+* What does module.exports do?
+* Why does method="post" even though we are updating (vs. creating) something?
+* Why do we use promises and callbacks when calling methods on a Mongoose model?
 
-```bash
-$ git checkout express-mongoose-solution
-```
+## Homework
+
+You should now be able to complete the [second part of the YUM homework](https://github.com/ga-wdi-exercises/yum#part-ii-add-express).
```

Comment:
They were told that they learned enough to do Part I after Friday afternoon's class. With this class, they should now be able to do Part II. Whether they've actually done Part I at this point, I don't know.